api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleService.java

```diff
@@ -98,16 +98,15 @@ public void savePaymentCycle(PaymentCycle paymentCycle) {
     }
 
     /**
-     * Update and saves the payment cycle with the given calculation and card balance.
+     * Update and saves the payment cycle with the given card balance.
      *
      * @param paymentCycle       payment cycle to update
-     * @param paymentCycleStatus payment cycle status
      * @param cardBalanceInPence card balance in pence
      */
-    public void updatePaymentCycle(PaymentCycle paymentCycle, PaymentCycleStatus paymentCycleStatus, int cardBalanceInPence) {
+    public void updatePaymentCycleCardBalance(PaymentCycle paymentCycle, int cardBalanceInPence) {
```

Comment:
The `updatePaymentCycleCardBalance` methods need refactoring as the overloading is confusing (but see patch attached above)

Reply:
That was a refactoring error, this has been fixed now.